packages/meter/package.json

```diff
@@ -62,7 +62,7 @@
         "@spectrum-web-components/shared": "^0.38.0"
     },
     "devDependencies": {
-        "@spectrum-css/progressbar": "^3.0.61"
+        "@spectrum-css/progressbar": "^3.0.71"
```

Comment:
This is at `3.0.73` do we want to try to grab the latest before moving this forward?

Reply:
yes sure!